packages/playground/package.json

```diff
@@ -48,6 +48,7 @@
     "@rjsf/core": "^4.0.0",
     "@rjsf/fluent-ui": "^4.0.0",
     "@rjsf/material-ui": "^4.0.0",
+    "@rjsf/primereact": "https://gitpkg.now.sh/dchenk/react-jsonschema-form/packages/primereact?8a7eb51f2f92d691dd2901476c54dd88245fb9d3",
```

Comment:
I added this to make sure that the playground works, and the version here can simply be updated upon the next release just like all the other `@rjsf/*` packages.

Reply:
Why lock it down to a specific version?